tests/support/helpers.py

```diff
@@ -1565,3 +1566,14 @@ def this_user():
     if salt.utils.is_windows():
         return salt.utils.win_functions.get_current_user(with_domain=False)
     return pwd.getpwuid(os.getuid())[0]
+
+
+def dedent(text, linesep=os.linesep):
+    '''
+    A wrapper around textwrap.dedent that also sets line endings.
+    '''
+    if isinstance(text, six.text_type):
+        linesep = salt.utils.stringutils.to_unicode(linesep)
```

Comment:
This needs to be `salt.utils.to_unicode(linesep)`, this function wasn't moved until 2018.3 and this PR has been opened against the 2017.7.8 release branch.